/**:
  ros__parameters:
    tf_message_filter_target_frame: "laser"
    scan_chain:
      scan_shadows_filter:
        params:
          min_angle: 10.0
          max_angle: 170.0
          neighbors: 20
          window: 1
      radius_search_filter:
        params:
          neighbor_number: 3
          threshold_number: 3
          threshold_ratio: 0.10
      laser_scan_range_filter:
        params:
          use_message_limits: false
          lower_threshold: 1.0
          lower_replacement_value: NaN
          upper_threshold: 9.0
          upper_replacement_value: 9.0
      laser_scan_box_filter:
        params:
          box_frame: laser
          min_x: 2.0
          max_x: 999.0
          min_y: 2.0
          max_y: 999.0
          min_z: -999.0
          max_z: 999.0
